website/docs/r/policy_project.html.markdown

```diff
@@ -19,7 +19,6 @@ resource "nsxt_policy_project" "test" {
   description         = "Terraform provisioned Project"
   short_id            = "test"
   tier0_gateway_paths = ["/infra/tier-0s/test"]
-
```

Comment:
Is this accidental or did you mean doing something else?

Reply:
just cosmetics - removing unneeded rows